AirCasting/SDSync/ViewsFlow/SDSyncingAB/SyncingABView.swift

```diff
@@ -32,12 +32,12 @@ struct SyncingABView<VM: SDSyncViewModel>: View {
         .padding()
         .background(navigationLink)
         .onReceive(viewModel.isSyncCompleted, perform: { isConnected in
-            viewModel.presentNextScreen = isConnected
+            viewModel.continueSyncFlow(value: isConnected)
```

Comment:
why is it even here instead of inside the viewmodel? It all can be done internally and set `presentNextScreen` and `failedSyncAlertPresent` from there.